### PR TITLE
Run3-hcx277 Make the other shower library producer to work for HF

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Make the other shower library producer using a standalone geometry setup (as done in HF TestBeam) to work for HF

#### PR validation:

Use the test workflows in SimG4CMS/ShowerLibraryProducer/test/python

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HcalCommonData/data/hcalforwardshower/v2/hcalforwardshower.xml [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+    0., 0., 0.
+  </Vector>
+</Algorithm>
+
+<SpecParSection label="hcalforwardshower.xml" eval="true">
+  <SpecPar name="fibre">
+    <PartSelector path="//HFFibre.*"/>
+    <Parameter name="SensitiveDetector" value="FiberSensitiveDetector" eval="false"/>
+    <Parameter name="ReadOutName" value="FibreHits" eval="false"/>
+    <Parameter name="rTable" value="12.50*cm"/>
```
Comment (cvuosalo) [inline Geometry/HcalCommonData/data/hcalforwardshower/v2/hcalforwardshower.xml:159]: Don't you want the numeric parameters to have `eval="true"`?

**Comment by cvuosalo** [comment]
New XML files should be put into `v1` directories.

File: Geometry/HcalCommonData/data/hcalforwardshower/v2/hcalforwardshower.xml (hunk 1 above)
Comment (bsunanda) [reply Geometry/HcalCommonData/data/hcalforwardshower/v2/hcalforwardshower.xml:159]: SpecParSection has eval="true" which is valid for all parts of spectra except some parameters which are string and has eval="false"

Comment (cvuosalo) [reply Geometry/HcalCommonData/data/hcalforwardshower/v2/hcalforwardshower.xml:159]: OK, thanks.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**